lib/trino-filesystem-gcs/src/main/java/io/trino/filesystem/gcs/GcsStorageFactory.java

```diff
@@ -81,7 +95,20 @@ public Storage create(ConnectorIdentity identity)
             if (projectId != null) {
                 storageOptionsBuilder.setProjectId(projectId);
             }
-            return storageOptionsBuilder.setCredentials(credentials).build().getService();
+            // Note: without uniform strategy we cannot retry idempotent operations.
```

Comment:
Without uniform strategy, the retries do not affect create operations so the tests would still fail, this strategy allows retrying all operations, since trino-filesystem api takes care of idempotency for create.